packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -218,8 +216,8 @@ const MetadataLabeling = (props: Props) => {
                 props.defaultVisibleValue
             )}
 
-            {((labelingAvailable && !props.payload.value) ||
-                (!labelingAvailable && device?.connected)) && (
+            {((!labelingAvailable && !props.payload.value) ||
+                (labelingAvailable && !props.payload.value)) && (
```

Comment:
This condition can be simplified to `{!props.payload.value && (`. Also for future reference, starting a condition with a lot of special characters makes it hard to read (in this case 4: `{((!`).

Reply:
it indeed looks retarded when looking at it after 3 weeks. so retarded that it is making me suspicions why I wrote it like this... hmm...